accessing-data/record-webhooks.mdx

```diff
@@ -129,8 +129,94 @@ Body:
       record_slug: 'ffd928c3-3459-43fb-a2e4-3f1b3ad5fc1f',
       validation_type: 'error',
       column_name: 'email',
-      issue_description: 'Email already exists and must be unqiue.'
+      issue_description: 'Email already exists and must be unique.'
     }]
   }
 }
-```
\ No newline at end of file
+```
+
+## Record Webhook Implementation Example
+
+In this example using Javascript and Express, we create an endpoint in your backend that initially receives the token `http_x_fuse_signature_256` containing the signature generated by Fuse and submitted to you. \
+You can use this signature to confirm the authenticity of the data, then, compare the signature with a generated one using the payload and your secret key.
+
+After that, we start validating the data, creating an empty array to store the possible issues, retrieving the `import_slug` for further usage and looping through the received data.
+
+In this step, you can do any validation you may need with the data, such as checking property existence in your backend, API calls, comparing values, etc. \
+For this example, we have two mock functions to simulate the data being validated: `validateIdExistsInTheBackend` and `validateEmailUniquenessInServer`.
+
+If you need to create some issue to be shown in the UI after the webhook is processed, you can use this structure below:
+
+```javascript
+{
+  import_slug: 'f50df76a-e5b9-4784-80ba-6ef27948e2ba',
+  record_slug: 'cb28e820-8af5-4f3a-b01b-87f2eb3f5014',
+  validation_type: 'error',
+  column_name: 'id',
+  issue_description: 'An error occurred for the ID column.',
+}
+```
+
+- `import_slug` can be accessed through `req.body.import_slug`.
```

Comment:
Describe what the parameters are.

Reply:
![image](https://github.com/flatironsdevelopment/fuse-docs/assets/50498997/e2f08cec-b978-47a2-8d3e-eb22c65ae38f)
Something like this? @mikefrederick